### DISCUSSION TITLE
documentation improvement suggestions/requests

### DISCUSSION THREAD

**Discussion by randyg503** [goauthentik/authentik#2856] [General]
A couple stale documentation areas to consider updating in the interest of making further forward progress on the Home Assistant application integration...

On the authentik web site, under Integrations > Applications > Home-Assistant, the Warning section indicates that "you might run into CSRF errors..." which goes on to refer to a Closed authentik GitHub issue.  Perhaps the section might also/instead reference an Open Home-Assistant GitHub issue dealing with same (if one doesn't exist, please consider creating one).

Also, the second note on the aforementioned page references a Closed (and seemingly rejected) Home-Assistant Pull Request and goes on to indicate that "...when this PR is merged, this will no longer be necessary."  Perhaps this outdated note should be updated to reflect the current PR status and/or refer to the relevant question/discussion occurring on the Home Assistant forums (as user 'frenck' suggests); again, if a question/discussion doesn't yet exist, please consider creating it.

Thank you!

**Comment by BeryJu** [comment]
See https://github.com/goauthentik/authentik/commit/4a9a19eacb43b819207a2ed6e836b9111861fe9a

The first part is caused by service workers in general, home assistant just happens to use them, so it's not really it's fault

I removed the second part since it's not really relevant anymore, Home assistant have stated their opinion and I'm tried of fighting it, and it's not really something that should be included in the docs

**Comment by randyg503** [comment]
Thank you for freshening up the docs, much appreciated!  

Something further to consider...it may be useful to preface the integration page in question with a statement indicating that 'Home Assistant currently does not support custom auth providers (e.g. authentik) and thus the authentik Home Assistant integration in its current form is a "workaround"'.  Possibly referring the reader to 'https://developers.home-assistant.io/docs/auth_auth_provider/' for additional information as provided by the Home Assistant devs.

Anyway, keep up the great work, thank you!

**Reply by randyg503** [reply]
...alternatively, a grid containing applications (y-axis) along with auth provider type technologies supported (x-axis) might be another useful macro-level tool to give the reader a good understanding of the landscape (albeit, at the expensive of keeping it updated)
